src/planscape/plan/views.py

```diff
@@ -315,6 +318,37 @@ def get_project(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def delete_projects(request: HttpRequest) -> HttpResponse:
+    try:
+        # Check that the user is logged in.
+        owner = _get_user(request)
+
+        body = json.loads(request.body)
+        project_ids = body.get('project_ids', None)
+        if project_ids is None or not (isinstance(project_ids, list)):
+            raise ValueError("Must specify project_ids as a list")
+
+        projects = [Project.objects.get(id=project_id) for project_id in project_ids]
+        
+        # Check that the user owns the projects
+        for project in projects:
+            if project.owner != owner:
+                raise ValueError(
```

Comment:
should we partially fail the request and still delete the projects that the user owns?

Reply:
Hmm, if the request includes IDs that the logged in user doesn't own, that suggests something went wrong in the frontend (they shouldn't be able to issue such a request from the UI). We shouldn't receive such a request to begin with, so I'm inclined to reflect the overall error state by failing the request completely. WDYT?

Reply:
this SGTM. thanks!

Reply:
Resolved offline; we won't do partial deletes.